app/lib/features/space/pages/chats_page.dart

```diff
@@ -43,13 +44,43 @@ class SpaceChatsPage extends ConsumerWidget {
                 child: Row(
                   mainAxisAlignment: MainAxisAlignment.end,
                   children: [
-                    IconButton(
-                      onPressed: () => context.pushNamed(
-                        Routes.createChat.name,
-                        queryParameters: {'spaceId': spaceIdOrAlias},
-                        extra: 1,
+                    PopupMenuButton(
+                      icon: Icon(
+                        Atlas.plus_circle,
+                        color: Theme.of(context).colorScheme.neutral5,
                       ),
-                      icon: const Icon(Atlas.plus_circle_thin),
+                      iconSize: 28,
+                      color: Theme.of(context).colorScheme.surface,
+                      itemBuilder: (BuildContext context) => <PopupMenuEntry>[
+                        PopupMenuItem(
+                          onTap: () => context.pushNamed(
+                            Routes.createChat.name,
+                            queryParameters: {'spaceId': spaceIdOrAlias},
+                            extra: 1,
```

Comment:
what is this `extra: 1` for?

Reply:
Not really aware about this param.  May be @gtalha07 can give better idea here.

Reply:
its a index of pages for ```PageView.builder()```, used for setting initial page of chat create sheet.

Reply:
so it isn't needed...
```suggestion
```